docs/userGuide/syntax/attributes.mbdf

```diff
@@ -0,0 +1,20 @@
+## Markdown-it-attributes
+
+Add classes, identifiers and attributes to your markdown with {.class #identifier attr=value attr2="spaced value"} curly brackets
+
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+```
+# header {.style-me}
+paragraph {data-toggle=modal}
+```
+</span>
+<span id="output">
+
+```
+<h1 class="style-me">header</h1>
+<p data-toggle="modal">paragraph</p>
+```
+</span>
+</includes>
```

Comment:
netlify preview seems to be broken

should we place this under the `formatting contents` section as well? since its related to markdown content

Reply:
> netlify preview seems to be broken

Trying to figure out why. 

> should we place this under the formatting contents section as well? since its related to markdown content

Placing there. PR is not ready, forgot to Draft it.